baybe/recommenders/pure/base.py

```diff
@@ -32,20 +32,30 @@ class PureRecommender(ABC, RecommenderProtocol):
     """Allow to make recommendations that were measured previously.
     This only has an influence in discrete search spaces."""
 
+    allow_recommending_pending_experiments: bool = field(default=False, kw_only=True)
+    """Allow `pending_experiments` to be part of the recommendations. If set to `False`,
+    the corresponding points will be removed from the candidates. This only influences
+    recommendations for purely discrete search spaces."""
```

Comment:
Feel free to adjust the wording of the last sentence but make it consistent across all flags.

Reply:
adjusted